curriculum/challenges/english/01-responsive-web-design/applied-accessibility/improve-accessibility-of-audio-content-with-the-audio-element.english.md

```diff
@@ -35,7 +35,7 @@ Time to take a break from Camper Cat and meet fellow camper Zersiax (@zersiax),
 tests:
   - text: Your code should have one <code>audio</code> tag.
     testString: assert($('audio').length === 1);
-  - text: Make sure your <code>audio</code> element has a closing tag.
+  - text: Your <code>audio</code> element should have a closing tag.
```

Comment:
Why not?
```suggestion
  - text: The <code>audio</code> element should have a closing tag.
```
as in https://github.com/freeCodeCamp/freeCodeCamp/pull/36860/files#diff-8d377b9fd2a899b7d2d08753b09a314cR29 ?

Reply:
The difference is in the one you referenced, the anchor element was moved, so the user did not add it.  I tried to use `The` if the element was pre-existing in the challenge seed and `Your` if the instructions told the user to add a new element.

Reply:
Ah, alright